app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkController.kt

```diff
@@ -77,7 +77,7 @@ class DefaultBookmarkController(
         val flags = EngineSession.LoadUrlFlags.select(EngineSession.LoadUrlFlags.ALLOW_JAVASCRIPT_URL)
         openInNewTabAndShow(
             item.url!!,
-            true,
+            false, // See https://github.com/mozilla-mobile/fenix/issues/13336
```

Comment:
No need to add a reference to the issue here. The commit message contains a reference to it already.

```suggestion
            false,
```